### PR TITLE
Add github actions for deploy to Jenkins

### PR DESCRIPTION
**Pull request by jendiamond**
@jhriv This/Jenkins needs 3 token names 
1. user
2. host
3. api

### CODE REVIEW

**Comment by jhriv** [comment]
Jenkins Tokens have been added.

File: .github/workflows/on-merge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    autoupdate:
+        name: Update outstanding PRs
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v1
+        - name: update all prs
+          uses: maxkomarychev/pr-updater-action@92cb6e15dc3c4b9b148b65eef32836bb428587b9  # v1.0.1
+          with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+        - name: Jenkins webhook
```
Comment (andrewbenedictwallace) [inline .github/workflows/on-merge.yml:18]: This should be done in its own job, so we can keep track

**Review by andrewbenedictwallace — COMMENTED** [review]
Looks great, but I think we should make a few housekeeping changes so it's easier to keep track of once we merge - see the individual comments

File: .github/workflows/on-merge.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Post-merge chores
```
Comment (andrewbenedictwallace) [inline .github/workflows/on-merge.yml:1]: Just use the name "On merge" - I've used names that didn't exactly match the filename and regretted it when I was trying to read things in the github log.

File: .github/workflows/scheduled-build.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Builds every day at 2:30pm
```
Comment (andrewbenedictwallace) [inline .github/workflows/scheduled-build.yml:1]: `name: Scheduled build`

File: .github/workflows/tag-build.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+name: Build with tag
```
Comment (andrewbenedictwallace) [inline .github/workflows/tag-build.yml:1]: Maybe change the filename to `on-release.yml` and use `name: On release`?

File: .github/workflows/on-merge.yml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+on:
+    push:
+        branches: 
+            - main
+
+jobs:
+    autoupdate:
+        name: Update outstanding PRs
```
Comment (andrewbenedictwallace) [inline .github/workflows/on-merge.yml:10]: again, match the name and the id (in this case it's `autoupdate`, but maybe `update-pull-requests` would be better?

**Review by jhriv — APPROVED** [review]

**Pull request merged**